lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -129,25 +110,27 @@ function runLighthouse(url, configPath, isDebug) {
 const cli = yargs
   .help('help')
   .describe({
-    'config-path': 'The path to the config JSON file',
-    'expectations-path': 'The path to the expected audit results file',
+    'smoke-id': 'The id of the smoke test to run',
```

Comment:
honestly at this point I don't see a reason we couldn't send the entire config/expectations as a stringified argument, this halfway point where you have to resolve everything ahead of time and then it still has to know how to lookup the definitions feels weird

Reply:
agreed ... So it's:

1. revert back to config/expectation paths and use tmp files as mentioned before
1. pass objects via CLI args

Is this smokehouse.js script _only_ meant to be used programmatically via run-smoke.js? I'm not sure if others use it directly. If not, then we can change it without breaking anyone's workflow and do option 2.

Reply:
Oh, `smokehouse.js` still needs a file path for the config, since it calls `lighthouse-cli/index.js --config-path=...`. Seems like we should just stick to paths since we can't change that.

Reply:
But to further complicate things, writing the expectations to a temporary file means replacing/reviving RegExps.

Reply:
OK, b/c of all the above I think this should just remain "smoke-id".